### DISCUSSION TITLE
Covariance Matrices in ACE Format

### DISCUSSION THREAD

**Discussion by sombrereau** [njoy/NJOY21#164] [General]
I see that the covariance matrices for WHISPER are distributed with what looks to be ACE format for covariance matrices. Is there a good example of how to do this in the NJOY manual or an example problem?

@bckiedrowski

**Comment by nathangibson14** [comment]
Hi @sombrereau,

The ACE-formatted covariance data is not made through NJOY.  It was an ad-hoc solution for Whisper.  We are working toward a covariance library for Whisper and others in a new format, based loosely on GNDS.  You should reach out to the Whisper/MCNP team if you have a need for covariances in this format.  From the NJOY side, we probably have tools that could be used to make these files with a fair amount of effort, but I don't think that's the solution you're looking for.

--Nathan

**Reply by sombrereau** [reply]
@nathangibson14  -- that answers my real question: "is this something NJOY could do out of the box"; the answer being no
